selftests/functional/test_basic.py

```diff
@@ -50,6 +50,17 @@ def test(self):
         self.log.info(hello())
 '''
 
+INVALID_PYTHON_TEST = '''
+from avocado import Test
+
+class MyTest(Test):
+
+    non_existing_variable_causing_crash
+
+    def test(self):
```

Comment:
Result will not contain `test` as the test name. It will contain the original test name function. I can check if we have `RESULTS    : PASS 0 | ERROR 1` in stdout or something like that. What do you think?